src/ScriptBuilder/CSharpCounterWriter.cs

```diff
@@ -49,45 +48,18 @@ public static void Create()
         var counterCreationCollection = new CounterCreationDataCollection(Counters);
         try
         {{
-            var install = false;
             var categoryName = ""NServiceBus"";
             if (PerformanceCounterCategory.Exists(categoryName))
             {{
-                var category = PerformanceCounterCategory.GetCategories().Single(x => x.CategoryName == categoryName);
-                var existingCounters = category.GetCounters();
-
-                if (existingCounters.Length != counterCreationCollection.Count)
-                {{
-                    install = true;
-                }}
-                else
-                {{
-                    foreach (var counter in Counters)
-                    {{
-                        var foundCounter = existingCounters.FirstOrDefault(c => c.CounterName == counter.CounterName);
-                        var found = foundCounter?.CounterName == counter.CounterName | foundCounter?.CounterType == counter.CounterType | foundCounter?.CounterHelp == counter.CounterHelp;
-                        if (!found)
-                        {{
-                            install = true;
-                        }}
-                    }}
-                }}
+                PerformanceCounterCategory.Delete(categoryName);
             }}
 
-            if (install)
-            {{
-                if (PerformanceCounterCategory.Exists(categoryName))
-                {{
-                    PerformanceCounterCategory.Delete(categoryName);
-                }}
-
-                PerformanceCounterCategory.Create(
-                    categoryName: categoryName,
-                    categoryHelp: ""NServiceBus statistics"",
-                    categoryType: PerformanceCounterCategoryType.MultiInstance,
-                    counterData: counterCreationCollection);
-                PerformanceCounter.CloseSharedResources();
-            }}
+            PerformanceCounterCategory.Create(
+                categoryName: categoryName,
+                categoryHelp: ""NServiceBus statistics"",
+                categoryType: PerformanceCounterCategoryType.MultiInstance,
```

Comment:
We create a single perf counter for Processing Time and then created an instance for each running endpoint. The same for each metric. (I believe this is how it is meant to work but I haven't confirmed it)